modules/tinymce/src/core/main/ts/content/ContentTypes.ts

```diff
@@ -27,6 +27,11 @@ export interface SetContentArgs {
   no_selection?: boolean;
 }
 
+export interface SetContentResult {
+  content: Content;
+  html: string;
```

Comment:
```suggestion
  readonly content: Content;
  readonly html: string;
```

As this appears to be an internal only type we should make sure the properties are readonly as well.